packages/cli/test/integration/commands/credentials.cmd.test.ts

```diff
@@ -23,6 +23,7 @@ beforeAll(async () => {
 	mockInstance(InternalHooks);
 	mockInstance(LoadNodesAndCredentials);
 	await testDb.init();
+	await oclifConfig.load();
```

Comment:
I tried it out and saw that it failed without that now. So I guess this was called implicitly before.

For educational purposes: How did you find out that this was the missing piece? I couldn't find anything in the docs or the changelog about this, nor in the issues.

Reply:
Their docs recommend executing commands like `Command.run(argv)`, which [in the code](https://github.com/oclif/core/blob/main/src/command.ts#L173) does `await config.load()`, `await command.init()`, and `await command.run()`. I just copied that bit over.